cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -274,6 +276,187 @@ sleep INF
 		})
 	})
 
+	Context("Rootless vhostnet", func() {
+		var sriovDevice *sriovv1.InterfaceExt
+		var node string
+		var isCUDDisabled bool
+		var tapIface string
+		var tapVlanIface100 string
+		var tapVlanIface200 string
+		var netAnnotationPodTX string
+		var netAnnotationPodRX string
+		var dpdkResource1 string
```

Comment:
Can we define it as constant? It looks like we assign the same value for these var in BeforeEach multiple times.
Same for:
dpdkResource2
tapNetwork
tapIface 
tapNetwork

Reply:
it is only done once and it's more to follow the convention of ginkgo while keeping these vars local